src/aiohabit/providers/aws.py

```diff
@@ -213,7 +213,7 @@ def to_host(self, provisioning_result):
             host_info.get("id"),
             host_info.get("name"),
             host_info.get("addresses"),
-            host_info.get("status"),
+            STATUS_MAP[host_info.get("status")],
```

Comment:
The STATUS_MAP doesn't cover all statuses. So we might want something similar as in OpenStack provider where is  `status = STATUS_MAP.get(provisioning_result.get("status"), STATUS_OTHER)`